packages/cli/src/commands/plugins.ts

```diff
@@ -0,0 +1,112 @@
+import { GluegunToolbox } from "gluegun";
+
+module.exports = {
+  name: "plugins",
+  hidden: true,
+  run: async (toolbox: GluegunToolbox) => {
+    const {
+      template: { generate },
+      print: { success, spin },
+    } = toolbox;
+
+    const isCIrun = toolbox.parameters.options.ci;
+    const inputParameters = toolbox.inputParameters;
+    if (!isCIrun) {
+      toolbox.print.info(
+        "Please provide username and password for plugins authentication. You can leave the fields empty if connecting to demo instance or if you don't need plugins :)"
+      );
+      const shopwareUsernameQuestion = !inputParameters.username && {
+        type: "input",
+        name: "username",
+        message: "Shopware admin username:",
+      };
+      const shopwarePasswordQuestion = !inputParameters.password && {
+        type: "password",
+        name: "password",
+        message: "Shopware admin password:",
+      };
+
+      const devModeQuestion = !inputParameters.devMode && {
+        type: "confirm",
+        name: "devMode",
+        message:
+          "Do you want to allow dev mode for slots? (Don't use for production!)",
+      };
+
+      const answers = await toolbox.prompt.ask([
+        shopwareUsernameQuestion,
+        shopwarePasswordQuestion,
+        devModeQuestion,
+      ]);
+      Object.assign(inputParameters, answers);
+    }
+
+    const allowDevMode = !!inputParameters.devMode;
+
+    await toolbox.loadPluginsAssets();
```

Comment:
in the future, we should prevent some possible timeouts/errors during the downloading and extracting/processing the cms components from SW6. I believe it can be even 20 plugins maybe more with a considerable size.

Reply:
sure, for now, it has loading indicator with CLI, we'll add something here when we will face real issue to test :)